addons/account/static/src/js/bills_tree_upload.js

```diff
@@ -25,6 +25,8 @@ odoo.define('account.upload.bill.mixin', function (require) {
         },
 
         _onFileUploaded: function () {
+            // Reset the file input, allowing to select again the same file if needed
+            this.$('.o_vendor_bill_upload .o_input_file').val('');
```

Comment:
Wouldn't it be more logical to do the cleanup at the end of this function, after the `do_action` is done?
Like in a `try/catch/finally` structure

Reply:
I moved it into the _rpc catch() to only reset if there is an error. Otherwise, it would redirect with the do_action so it does not matter anyway.